### ISSUE TITLE
[$75] Copilot updates

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/topcoder-x-ui#332]
After #328 is done, we'll make some additional changes:

### Payment amount field

On the `Manage Project` page, add a field for Copilot Payment Amount, default it to 40, editable by admins only

### Assignee vs copilot

There are certain tickets where the assignee *is* the copilot.  In that case, we want the copilot ticket to *not* be added because the copilot would get double-paid in that case.

### Payment comments

On the git comment where the task is Paid, add a “receipt” with no $$ amounts:

```
*Payments Complete*
Winner: lieutenantroger
Copilot: ghostar2020
```

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30134520 has been created for this ticket.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by jmgasper** [comment]
Contest https://www.topcoder.com/challenges/30134520 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for ghostar via Topcoder X```

**Comment by afrisalyp** [comment]
@jmgasper 

> Payment comments

If Create Copilot Payments option is disabled. Will we still print the copilot in the comment?

**Comment by jmgasper** [comment]
@afrisalyp - No, if the payments option is disabled, we won't print the copilot in the comment.

**Comment by afrisalyp** [comment]
@jmgasper 

Ok, thanks.

> Payment amount field

How's the amount value affect the closing process?

**Comment by jmgasper** [comment]
@afrisalyp - I thought we set that value in our code, but as I look at it, I think it's being set by the platform.  Let me confirm with Topcoder.  We may need to wait on this until we merge in the V5 API changes.

**Comment by afrisalyp** [comment]
@jmgasper 

Okay sure.
A PR submitted for `Assignee vs copilot` and `Payment comments` items.

**Comment by jmgasper** [comment]
Closing for now.  Once I get clarification on the payment amount (may be v5-only), then we'll open up a new ticket.

**Comment by jmgasper** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30134520<br/><br/>```This is an automated message for ghostar via Topcoder X```
